external/fv3fit/fv3fit/emulation/layers/architecture.py

```diff
@@ -112,6 +115,83 @@ def get_config(self):
         return config
 
 
+class RNN(tf.keras.layers.Layer):
```

Comment:
Not essential, but could this be turned into a function returning `tf.keras.Sequential`. e.g. 

```
tf.keras.Sequential([SimpleRNN(depth, go_backwards, channels) for i in range(depth)])
```

`Sequential` is a instance of `Layer` too.

Reply:
I am thinking that `go_backwards` and also depending upon the state output would end up with alternating directional dependence, and the output vertical level orientation would depend on the depth in this example.  That's why I opted to use the `tf.reverse` instead.

Reply:
ok...not a big deal. I wouldn't expect SimpleRNN to flip the direction of the output though.